assets/training/model_evaluation/src/evaluators/evaluators.py

```diff
@@ -617,16 +617,32 @@ def evaluate(self, y_test, y_pred, **kwargs):
         """
         #  dataframe with 2 columns predictions and predictions appended to the conversation
         if len(y_pred.columns) > 1:
```

Comment:
Are we validating y_pred to be of the correct type?

Reply:
Yes, y_pred is always a dataframe. We take care of that while loading the data